# core_oss_framepointer.patch by Steve Kowalik
#
# Compile snd-pcm-oss.o with -fno-omit-frame-pointer.
Index: alsa-driver-1.0.15/alsa-kernel/core/oss/Makefile
===================================================================
--- alsa-driver-1.0.15.orig/alsa-kernel/core/oss/Makefile	2007-10-15 10:45:06.000000000 +0200
+++ alsa-driver-1.0.15/alsa-kernel/core/oss/Makefile	2007-10-21 03:36:14.000000000 +0200
@@ -3,6 +3,8 @@
 # Copyright (c) 1999 by Jaroslav Kysela <perex@perex.cz>
 #
 
+EXTRA_CFLAGS += -fno-omit-frame-pointer
+
 snd-mixer-oss-objs := mixer_oss.o
 
 snd-pcm-oss-y := pcm_oss.o
